### DISCUSSION TITLE
Allow manually setting power consumption for each miner/algo

### DISCUSSION THREAD

**Discussion by ch1rish** [RainbowMiner/RainbowMiner#2633] [Ideas]
The power readouts are extremely inaccurate and kneecap the full potential of RM. If we could manually set our power draw for each algo for each miner, we could provide accurate power consumption data that would make RM the most accurate profit switching product on the market.

**Comment by RainbowMiner** [comment]
I will think about that. At the moment we use the miner's APIs to read out the power consumption, if possible. If not, we use available tools.
